libirods_rule_engine_plugin-elasticsearch.cpp

```diff
@@ -136,12 +141,16 @@ namespace
 			using tcp = net::ip::tcp;
 
 			// This lambda encapsulates the HTTP logic that's independent of the type of stream.
-			const auto construct_and_send_http_request = [_verb, _target, _body](auto& _stream) {
+			const auto construct_and_send_http_request = [_verb, _target, _body, use_tls](auto& _stream) {
 				http::request<http::string_body> req{_verb, _target, 11};
 				req.set(http::field::host, boost::asio::ip::host_name());
 				req.set(http::field::user_agent, "iRODS Indexing Plugin/" IRODS_PLUGIN_VERSION);
 				req.set(http::field::content_type, "application/json");
 
+				if (use_tls && !config->base64_encoded_creds.empty()) {
```

Comment:
You may have to expand some lines to see this, but `use_tls` is true when the ES host contains `https://`.

The README should mention that.

Reply:
Done